Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py

```diff
@@ -102,16 +102,20 @@ def find_words_contain_tokens(positive_tokens, words_to_token_maps):
     return positive_words
 
 
-if __name__ in ['__main__', '__builtin__', 'builtins']:
+def main():
     result = predict_phishing_words(demisto.args()['modelName'],
                                     demisto.args()['modelStoreType'],
                                     demisto.args().get('emailSubject', ''),
-                                    demisto.args().get('emailBody', ''),
+                                    demisto.args().get('emailBody', '') or demisto.args().get('emailBodyHTML', ''),
```

Comment:
The new input is intended for a raw html text? if so, Perhaps we should preprocess it?